tensorflow_datasets/core/features/video_feature.py

```diff
@@ -197,26 +185,80 @@ def to_json_content(self) -> Json:
         'ffmpeg_extra_args': self._extra_ffmpeg_args
     }
 
+  def _generate_video_html(self, images, framerate=24, encoding_format='mp4') -> str:
+    """Converts sequence of images into video string."""
+    imgs = len(str(len(images)))+1  # Find number of digits in len to give names.
+
+    with tempfile.TemporaryDirectory() as video_dir:
+      for i, img in enumerate(images):
+        f = os.path.join(video_dir, f'img{i:0{imgs}d}.png')
+        img.save(f, format='png')
+
+      ffmpeg_args = [self._ffmpeg_path, '-framerate', str(framerate), '-i',
+                    os.fspath(os.path.join(video_dir, f'img%0{imgs}d.png'))]
+
+      extra_ffmpeg_args = ['-vf', 'scale=128:128', '-vcodec', 'h264',
+                           '-pix_fmt', 'yuv420p', '-allow_sw', '1']
+      ffmpeg_stdin = None
+
+      output_pattern = os.path.join(video_dir, 'output.' + encoding_format)
+      ffmpeg_args += extra_ffmpeg_args
+      ffmpeg_args.append(output_pattern)
+      try:
+        _ffmpeg_run(ffmpeg_args, ffmpeg_stdin)
+
+        video = None
+        video_file = os.path.join(video_dir, 'output.mp4')
+        with tf.io.gfile.GFile(video_file, 'rb') as vf:
+          video = vf.read()
+
+        def write_buff(buff):
+          buff.write(video)
+        video_str = utils.get_base64(write_buff)
+        return (f'<video height="128" width="128" controls loop plays-inline>'
+                f'<source src="data:image/gif;base64,{video_str}"'
```

Comment:
Why is source `data:image/gif` ? Shouldn't this be video ? Is it why HTML5 was bugging ?

Reply:
Oh! Its a typo I made, thanks. No, I don't think its the reason for bugging the chrome support. It was most probably because chrome didn't support that codec.

Reply:
For reference, which codec wasn't supported ?

Reply:
Are you sure it wasn't the MIME ? From https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types

> **Important:** Browsers use the MIME type, not the file extension, to determine how to process a URL, so it's important that web servers send the correct MIME type in the response's Content-Type header. If this is not correctly configured, browsers are likely to misinterpret the contents of files and sites will not work correctly, and downloaded files may be mishandled.

Reply:
>Are you sure it wasn't the MIME ? From https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types

Yes, I did correct the MIME type, disabled the extra args and rechecked, it did not work, so pretty much sure that the codec wasn't supported.

> For reference, which codec wasn't supported ?

`mpeg` Please refer: https://stackoverflow.com/questions/17793150/default-ffmpeg-codec-when-nothing-is-specified